assets/models/system/bytetrack_yolox_x_crowdhuman_mot17-private-half/description.md

```diff
@@ -0,0 +1,92 @@
+
+> The above abstract is from MMTracking website. Review the <a href="https://github.com/open-mmlab/mmtracking/tree/v0.14.0/configs/mot/bytetrack" target="_blank">original-model-card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
+
+### Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-video-mutli-object-tracking-online-inference" target="_blank">video-multi-object-tracking-online-endpoint.ipynb</a>|[video-multi-object-tracking-online-endpoint.sh](https://github.com/Azure/azureml-examples/blob/yiyoulin/mot_examples/cli/foundation-models/system/inference/video-multi-object-tracking/video-multi-object-tracking-online-endpoint.sh)|
+
+### Finetuning samples
+
+Task|Use case|Dataset|Python sample (Notebook)|CLI with YAML
+|---|--|--|--|--|
+Video multi-object tracking|Video multi-object tracking|[MOT17 tiny](https://download.openmmlab.com/mmtracking/data/MOT17_tiny.zip)|<a href="https://aka.ms/azureml-video-multi-object-tracking-finetune" target="_blank">mmtracking-video-multi-object-tracking.ipynb</a>|[mmtracking-mot17tiny-mot.sh](https://github.com/Azure/azureml-examples/blob/yiyoulin/mot_examples/cli/foundation-models/system/finetune/video-multi-object-tracking/mmtracking-mot17tiny-mot.sh)|
+
+
+### Sample inputs and outputs (for real-time inference)
+
+#### Sample input
+
+```json
+{
+  "input_data": {
+    "columns": [
+      "video"
+    ],
+    "data": ["video_link"]
+  }
+}
+```
+
+Note: "video_link" should be publicly accessible urls.
+
+#### Sample output
+
+```json
+[
+  {
+    "det_bboxes": [
+      {
+        "box": {
+          "topX": 703.9149780273,
+          "topY": -5.5951070786,
+          "bottomX": 756.9875488281,
+          "bottomY": 158.1963806152
+        },
+        "label": 0,
+        "score": 0.9597821236
+      },
+      {
+        "box": {
+          "topX": 1487.9072265625,
+          "topY": 67.9468841553,
+          "bottomX": 1541.1591796875,
+          "bottomY": 217.5476837158
+        },
+        "label": 0,
+        "score": 0.9568068385
+      }
+    ],
+    "track_bboxes": [
+      {
+        "box": {
+          "instance_id": 0,
+          "topX": 703.9149780273,
+          "topY": -5.5951070786,
+          "bottomX": 756.9875488281,
+          "bottomY": 158.1963806152
+        },
+        "label": 0,
+        "score": 0.9597821236
+      },
+      {
+        "box": {
+          "instance_id": 1,
+          "topX": 1487.9072265625,
+          "topY": 67.9468841553,
+          "bottomX": 1541.1591796875,
+          "bottomY": 217.5476837158
+        },
+        "label": 0,
+        "score": 0.9568068385
+      }
+    ],
+    "frame_id": 0,
+    "video_url": "https://github.com/open-mmlab/mmtracking/raw/master/demo/demo.mp4"
```

Comment:
Minor: Maybe we can use "video_link" instead of an actual github url here, which will be inline with the input data.

Reply:
changed. :)